ivanti_security_controls/plugin.spec.yaml

```
@@ -797,3 +921,71 @@ actions:
         description: Information about a machine involved with the patch deployment
         type: machine_deploy_state
         required: true
+  create_patch_scan_template:
+    title: Create Patch Scan Template
+    description: Create a new patch scan template
+    input:
+      name:
+        title: Name
+        description: Name of the patch scan template
+        type: string
+        required: true
+        example: ExamplePatchScanTemplate
+      description:
+        title: Description
+        description: Description that explains the purpose of this patch scan template
+        type: string
+        required: false
+        example: Created by REST API
+      path:
+        title: Path
+        description: Path to the location of the machine group within the Patch Scan Templates list in the navigation pane
+        type: string
+        required: false
+        example: Lab\Servers
+      threadCount:
+        title: Thread Count
+        description: Specifies maximum number of machines that can be simultaneously scanned during one patch scan
+        type: integer
+        required: false
+        example: 1
+      patchGroupIds:
+        title: Patch Group IDs
+        description: The IDs of the patch groups to use
+        type: '[]integer'
+        required: true
+        example: 1
+    output:
+      patch_scan_template:
+        title: Patch Scan Template
+        description: Detailed information about the patch scan template
+        type: patch_scan_template
+        required: true
+  create_patch_group_and_add_cves:
+    title: Create Patch Group and Add CVEs
```

I know the ticket didn't have a title, we should probably just call this
```suggestion
    title: Create Patch Group
```